cowprotocol/cowswap
# Updated comfirmation modal to show executed order details

# Summary

[Second part](https://www.notion.so/cownation/Show-Me-the-Money-hackathon-0a1b1b3e6a1b4d4eb9a46f96136a1865?pvs=4#638f3d64c891445bb3c04e7c046374b7) of the surplus saga

This PR updates the confirmation modal to show order summary and surplus once the order is executed
Actually once the order is executed it will first show confirmation text as before and then after 4s it will show this new content

New content
<img width="889" alt="Screenshot 2023-04-12 at 15 40 39" src="https://user-images.githubusercontent.com/34926005/231477551-8dc1c968-e7f6-4bb7-97f3-36ace9e4c5ce.png">

### To test
1. post an order
2. when confirmation modal opens don't close it
3. wait until the order is executed
4. after 4s you should see this new content
